background.js

```diff
@@ -401,18 +401,18 @@ chrome.webRequest.onCompleted.addListener(function(details) {
       continue; // don't remove cookies
     }
     chrome.cookies.getAll({domain: domainVar}, function(cookies) {
-		for (var i=0; i<cookies.length; i++) {
-			var cookie_domain = cookies[i].domain;
-			// hold specific cookie(s) from remove_cookies domains
-			if ((cookie_domain in remove_cookies_select_hold) && remove_cookies_select_hold[cookie_domain].includes(cookies[i].name)){
-				continue; // don't remove specific cookie
-			}
-			// drop only specific cookie(s) from remove_cookies domains
-			if ((cookie_domain in remove_cookies_select_drop) && !(remove_cookies_select_drop[cookie_domain].includes(cookies[i].name))){
-				continue; // only remove specific cookie
-			}
-			chrome.cookies.remove({url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name});
-		}
+    for (var i=0; i<cookies.length; i++) {
+      var cookie_domain = cookies[i].domain;
+      // hold specific cookie(s) from remove_cookies domains
+      if ((cookie_domain in remove_cookies_select_hold) && remove_cookies_select_hold[cookie_domain].includes(cookies[i].name)){
+        continue; // don't remove specific cookie
+      }
+      // drop only specific cookie(s) from remove_cookies domains
+      if ((cookie_domain in remove_cookies_select_drop) && !(remove_cookies_select_drop[cookie_domain].includes(cookies[i].name))){
+        continue; // only remove specific cookie
+      }
+      chrome.cookies.remove({url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name});
```

Comment:
Just an object styling preference, however, I would suggest using [airbnb](https://github.com/airbnb/javascript#objects) to make code more readable. 
```suggestion
      chrome.cookies.remove({
          url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path,
          name: cookies[i].name
      });
```
or

```suggestion
      chrome.cookies.remove({ url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name });
```